lib/pbio/drv/uart/uart_debug_first_port.h

```diff
@@ -28,6 +32,13 @@ void pbdrv_uart_debug_init(void);
 
 #define pbdrv_uart_debug_init()
 
+static inline lwrb_t *pbdrv_uart_debug_get_ring_buffer(void) {
+    return NULL;
+}
+
 #endif // PBDRV_CONFIG_UART_DEBUG_FIRST_PORT
 
+// Convenient shorthand for pbdrv_uart_debug_printf.
+#define pb_log(...) pbdrv_uart_debug_printf(__VA_ARGS__)
```

Comment:
```suggestion
#define pbdrv_dbg(...) pbdrv_uart_debug_printf(__VA_ARGS__)
```

We use the `pb_` prefix for MicroPython code. 

And keeping debug in the name would be nice.

Reply:
Done.